docs/dev/guidelines/code-review.rst

```diff
@@ -0,0 +1,61 @@
+***********
+Code Review
+***********
+
+STEPS
+======
+
+0. Precondition -> only Developer
+==================================
+
+* Limit yourself to one functionality per pull request
+* Split up your task in multiple branches & pull requests if necessary
+* `Commit Early, Commit Often, Perfect Later, Publish Once <https://speakerdeck.com/lemiorhan/10-git-anti-patterns-you-should-be-aware-of>`_
+
+1. Start Implementation -> only Developer
+=========================================
+
+* `Open draft pull request. <https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request>`_ Allows for code related questions and discussions
+
+2. Implementation is "done" -> only Developer
+=============================================
+
+* Make sure all steps in the `Checklist <https://github.com/ls1intum/ArTEMiS/blob/develop/.github/PULL_REQUEST_TEMPLATE.md>`_ are completed
+* Add or update the "Steps for Testing" in the description of your pull request
+* Mark the pull request as `ready for review <https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request>`_
+* Add Stephan Krusche (@krusche) and Jan Philip Bernius (@jpbernius) and your review partner as reviewers to the pull request
```

Comment:
`@krusche` and `@jpbernius` seem to be added automatically already, so not sure if we have to list that here too.

Reply:
yes, we should remove this point